core/src/views/UnsupportedBrowser.vue

```diff
@@ -0,0 +1,191 @@
+import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
+import NcEmptyContent from '@nextcloud/vue/dist/Components/NcEmptyContent'
+import Web from 'vue-material-design-icons/Web'
+
+import { browserStorageKey } from '../utils/RedirectUnsupportedBrowsers.js'
+import { supportedBrowsers } from '../services/BrowsersListService.js'
+import browserStorage from '../services/BrowserStorageService.js'
+import logger from '../services/LoggerService.js'
+
+logger.debug('Supported browsers', { supportedBrowsers })
+
+export default {
+	name: 'UnsupportedBrowser',
+	components: {
+		Web,
+		NcButton,
+		NcEmptyContent,
+	},
+
+	data() {
+		return {
+			agents: {},
+		}
+	},
+
+	computed: {
+		isMobile() {
+			return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
+		},
+
+		/**
+		 * Filter out or include mobile/desktop browsers depending
+		 * on the current user platform/device
+		 */
+		filteredSupportedBrowsers() {
+			return supportedBrowsers.filter(browser => {
+				if (!browser) {
+					return false
+				}
+
+				if (this.isMobile) {
+					return this.isMobileBrowser(browser)
+				}
+				return !this.isMobileBrowser(browser)
+			})
+		},
+
+		formattedBrowsersList() {
+			const list = {}
+
+			// supportedBrowsers is generated by webpack at compilation time
+			this.filteredSupportedBrowsers.forEach(browser => {
+				const [id, version] = browser.split(' ')
+				if (!list[id] || list[id] < parseFloat(version, 10)) {
+					list[id] = parseFloat(version, 10)
+				}
+			})
+
+			return Object.keys(list).map(id => {
+				if (!this.agents[id]?.browser) {
+					return null
+				}
+
+				const version = list[id]
+				const name = this.agents[id]?.browser
+				return this.t('core', '{name} version {version} and above', {
+					name, version,
+				})
+			}).filter(entry => entry !== null)
+		},
+	},
+
+	async beforeMount() {
+		// Dynamic load big list of user agents
+		// eslint-disable-next-line node/no-extraneous-import
+		const { agents } = await import('caniuse-lite')
+		this.agents = agents
+	},
+
+	methods: {
+		t,
+		n,
+
+		// Set the flag allowing this browser and redirect to home
+		forceBrowsing() {
+			browserStorage.setItem(browserStorageKey, true)
+			window.location = generateUrl('/')
+		},
+
+		/**
+		 * Detect if the browserslist browser is a mobile one
+		 * https://github.com/browserslist/browserslist#query-composition
+		 *
+		 * @param {string} browser a valid browserlist browser. e.g `and_chr 90`
+		 */
+		isMobileBrowser(browser) {
+			browser = browser.toLowerCase()
+			return browser.includes('and_')
+				|| browser.includes('android')
+				|| browser.includes('ios_')
+				|| browser.includes('mobile')
+				|| browser.includes('_mob')
+				|| browser.includes('samsung')
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+.content-unsupported-browser {
+	display: flex;
+	justify-content: center;
+	width: 400px;
+	max-width: 90vw;
```

Comment:
```suggestion
	max-width: 80vw;
```
loogs better on very small  widths

Reply:
Ah right, the box-sizing is applied to content only! :see_no_evil: